src/helpers.js

```diff
@@ -12,6 +12,9 @@ governing permissions and limitations under the License.
 
 const jwt = require('jsonwebtoken')
 const debug = require('debug')('@adobe/aio-lib-ims-jwt')
+const fs = require('fs') // need promise
```

Comment:
why not fs-extra like everywhere else?

Reply:
fs extra is not a current dependency to the module, we just need to call fs.readFile so I feel it's a waiste to add the dependency for one method + there node 12 supports `const fs = require('fs').promises;` (node 10 too but outputs an experimental warning)